flang/include/flang/Evaluate/integer.h

```diff
@@ -150,7 +150,10 @@ class Integer {
           }
         }
       } else {
-        INT signExtension{-(n < 0)};
+        // Avoid left shifts of negative signed values (that's an undefined
+        // behavior in C++).
+        auto signExtension = std::make_unsigned_t<INT>(n < 0);
```

Comment:
Use braced initialization in Flang front-end code.

Reply:
Done